app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -86,9 +85,30 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
     }
 
     override fun populateDeparturesTable(departuresList: List<DepartureInformation>) {
+        setLoadingSpinner(false)
         departures.clear()
         departures.addAll(departuresList)
         viewAdapter.notifyDataSetChanged()
         recyclerView.smoothScrollToPosition(0);
     }
+
+    override fun createAlertMessage(alertMessage: String) {
+        val alertDialogBuilder = AlertDialog.Builder(this)
+        alertDialogBuilder.setMessage(alertMessage)
+        alertDialogBuilder.setNeutralButton("Ok") { _, _ -> null}
+        alertDialogBuilder.show()
+        setLoadingSpinner(false)
+    }
+
+    private fun setLoadingSpinner(visible: Boolean) {
```

Comment:
Maybe showLoadingSpinner would be a slightly better name, as it describes slightly better what this function actually does? What do you think?